examples/mcp2221_single_busio_i2c.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2022, The Blinka Authors.
+#
+# SPDX-License-Identifier: MIT
+import argparse
+import busio
+
+
+# Usage:
+# $ lsusb | grep Microchip
+# Bus 001 Device 082: ID 04d8:00dd Microchip Technology, Inc.
+# Bus 001 Device 083: ID 04d8:00dd Microchip Technology, Inc.
+# $ python mcp2221_single_busio_i2c.py -b 1 -d 83
+# I2C devices found:  ['0x70']
+
+
+# Bus and Device number can be gathered from `lsusb`
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-b", "--bus", type=int, required=True, help="USB bus number (base10)"
+)
+parser.add_argument(
+    "-d", "--device", type=int, required=True, help="USB device number (base10)"
+)
+args = parser.parse_args()
+
+# Need to be bytes, always function 2
+device_path = "{:04x}:{:04x}:02".format(args.bus, args.device).encode()
```

Comment:
This script doesn't work for me because of this line.  The paths to my devices look like `2-3.3:1.2`, whereas this will produce something like `0002:0003:02`.  Perhaps the example could be updated to print a list of available addresses, and prompt the user to select one.